challenges/objects-arrays.js

```diff
@@ -50,20 +77,35 @@ const graduates = [{"id":1,"first_name":"Cynde","university":"Missouri Southern
 
 Once you have the new array created, sort the universities alphabetically and log the result. */
 const universities = [];
+graduates.forEach(function(element) {
+  universities.push(`${element.university}`)
+  universities.sort();
+  return universities;
+})
 console.log(universities)
 
-/* Request 2: Create a new array called contactInfo that contains both first name and email of each student. 
+/* Request 2: Create a new array called contactInfo that contains both first 
+name and email of each student. 
 
 The resulting contact information should have a space between the first name and the email information like this: 
 Name email@example.com
 
 Log the result of your new array. */
 const contactInfo = [];
+graduates.forEach(function(chocolate) {
+  contactInfo.push(`${chocolate.first_name} ${chocolate.email}`);
+});
 console.log(contactInfo);
 
 
-/* Request 3: Find out how many universities have the string "Uni" included in their name. Create a new array called uni that contains them all. Log the result. */
+/* Request 3: Find out how many universities have the string "Uni" included in their 
+name. Create a new array called uni that contains them all. Log the result. */
 const uni = [];
+for(let i = 0; i < graduates.length; i++) {
+  if(graduates[i].university.includes(`Uni`) === true) {
```

Comment:
one more trick here is to take out the `=== true` 
since includes only returns true.